rpm_spec/securedrop-workstation.spec

```diff
@@ -0,0 +1,63 @@
+Name:		securedrop-workstation
```

Comment:
We need a more descriptive title here; perhaps `securedrop-workstation-dom0-config`.

Reply:
We can do this change, but it would be nicer to have it the same name of the tarball+github project. Any thoughts? If you insist, we can make the change :)